luigi/task.py

```diff
@@ -52,9 +52,9 @@ def namespace(namespace=None):
     Call to set namespace of tasks declared after the call.
 
     If called without arguments or with ``None`` as the namespace, the namespace
-    is reset, which is recommended to do at the end of any file where the
-    namespace is set to avoid unintentionally setting namespace on tasks outside
-    of the scope of the current file.
+    is reset to :py:attr:`Register._UNSET_NAMESPACE`., which is recommended to do
```

Comment:
Actually can we avoid mentioning `Register._UNSET_NAMESPACE`. I thought of it as an implementation detail of the simpler concept of "being unset". 

Perhaps the docs become more natural if we remove the  "If called without arguments or with ..." part and just say "you have to call this function without arguments at the end of any file it has been used. That is to ensure blah blah". Something like that. Just a suggestion.